artnet.py

```diff
@@ -88,6 +90,11 @@ def run(self):
         self._socket.close()
         self._dmx.artnet_status = 'offline'
     
+    @staticmethod
+    def run_render():
+        bpy.context.scene.dmx.render()
+        return (1.0/60.0)
```

Comment:
Could you elaborate on how the FPS is controlled, to keep it documented? Maybe on the future it might be interesting to constrain it through settings.
Does blender timer compensate for the time the `run_render` function takes? Or does it wait for a fixed time?

Reply:
In the future we might take some time to benchmark the FPS and understand if we need to compensate for that (in order to keep a steady fps), but for now it looks good!